checkov/kustomize/runner.py

```diff
@@ -275,8 +358,21 @@ def _parseKustomization(self, kustomize_dir: str) -> dict[str, Any]:
                 return {}
 
             if 'resources' in file_content:
-                logging.debug(f"Kustomization contains resources: section. Likley a base. {kustomization_path}")
-                metadata['type'] = "base"
+                resources = file_content['resources']
+
+                # We can differentiate between "overlays" and "bases" based on if the `resources` refers to a directory,
+                # which represents an "overlay", or only files which represents a "base"
+                resources_representing_directories = [r for r in resources if pathlib.Path(r).suffix == '']
+                if len(resources_representing_directories) > 0:
```

Comment:
```suggestion
                if resources_representing_directories:
```
no need to explicitly calculate the length here